### ISSUE TITLE
Magento Admin pages not fully loading and rendering when opened in the background

### ISSUE THREAD

**Issue by orbitrod** [magento/magento2#15504]
### Preconditions

1.  Magento Version 2.2.3
2. Chrome Version 66.0.3359.181 (Official Build) (64-bit)

### Steps to reproduce

1. Login to Magento Admin
2. Navigate to Products
3. Open multiple edit product links in new tabs
4. Wait a few seconds for all loading indicators to stop spinning for new tabs
5. Switch to new tabs that were opened

### Expected result

1. The new pages should fully load and render in the background.
2. When switching to the new tab the page should be fully loaded and rendered.

### Actual result

1. When switching to the new tabs, the pages are not ready to go.  They are not fully loaded and require a little more time for the product fields and information to populate.

![magento admin background loading_1](https://user-images.githubusercontent.com/13354171/40505208-1eaeffa4-5f59-11e8-8164-ffa9e572b94c.gif)

**Comment by orbitrod** [comment]
I have done some more testing and I believe that this behavior might be exclusive to Chrome.  Firefox and Edge do not behave this way.  They fully load pages in the background.

Does anyone know why Chrome might be behaving this way vs the other browsers?

**Comment by orbitrod** [comment]
Does anyone have any advice on how I can get these pages to load in the background?

**Comment by ghost** [comment]
@orbitrod, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by orbitrod** [comment]
Awesome, thank you for the update.

**Comment by shikhamis11** [comment]
@orbitrod I have found the same issue on  Firefox too. I believe this is not browser specific It must be due to Magento 2 UI js behavior. I will work on it to find the exact reason behind it.

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by orbitrod** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @orbitrod. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @orbitrod, here is your [Magento instance](https://i-15504-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-15504-2-3-develop.instances.magento-community.engineering/admin](https://i-15504-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by orbitrod** [comment]
Reproduced on 2.3.x

**Comment by MarcusWolschon** [comment]
> Firefox



> I have done some more testing and I believe that this behavior might be exclusive to Chrome. Firefox and Edge do not behave this way. They fully load pages in the background.
> 
> Does anyone know why Chrome might be behaving this way vs the other browsers?

Firefox clearly behaves the same.  (Magento 2.3.3)
I'm annoyed by this ever since I switched from Magento 1.9 to Magento 2.1 (and later 2.2 and 2.3).

**Comment by MarcusWolschon** [comment]
"Assignees: No one assigned" after 1.5 years?

**Comment by orbitrod** [comment]
@MarcusWolschon Glad to see you on this issue.  If we can get more people to confirm this problem, we might be able to get the attention of the developers to issue a fix.

**Comment by Grumag** [comment]
This poor design or a bug is still happening in 2.3.5-p1. Even more time is wasted when you do tracking on Sales > Orders page. 

I will give you an example.
1. Open an order to enter tracking information, once done click on Back button, page starts loading
2. Switch to another tab to, for example, get tracking for another order
3. Go make a coffee and get back after 15 minutes
4. Open the tab showing Orders... it hasn't loaded at all!!!!

I am not sure how this "feature" is still in Magento 2, after 4 years. Small businesses running M2 do not have unlimited amount of time. Why would you stop loading pages when a tab is switched? Those 5-7 seconds it takes to load the Order page I can do "other" stuff. Those seconds add up when you do 10 hrs a day  every in Magento 2. Countless hours are wasted waiting for page to load.

**Comment by mklooss** [comment]
just for the search folks:

you just have to add an template to the pages oder global for every adminhtml page with "$(window).trigger('blur');".

```html
<script type="text/javascript">
    require([
        'jquery'
    ], function ($) {
        window.setTimeout(function() {
            $(window).trigger('blur');
        }, 1000)
    });
</script>
```

**Comment by OvalMedia** [comment]
Yes, this is really annoying. Think of all the power users that have to work with magento every day.
I am just developing and I am already getting nervous twitches whenever I see a spinner on the screen.

**Comment by YiffyToys** [comment]
> just for the search folks:
> 
> you just have to add an template to the pages or global for every adminhtml page with "$(window).trigger('blur');".
> 
> tested with magento 2.3


Can you aleboate on that?
What does it do and what part of the issue does it work around?

**Comment by mklooss** [comment]
We take some time and research and found out the issue is more an Browser/JavaScript Issues.
"inactive Tabs" slow down the JavaScript Resources, some helpfull links:

- https://stackoverflow.com/questions/6032429/chrome-timeouts-interval-suspended-in-background-tabs
- https://bugzilla.mozilla.org/show_bug.cgi?id=652472
- https://superuser.com/questions/1500289/how-to-aggressively-throttle-background-tabs-in-firefox-using-dom-min-background

The solution on Stackoverflow with the sound file, just works on Chrome

**Comment by MarcusWolschon** [comment]
> We take some time and research and found out the issue is more an Browser/JavaScript Issues.

That's execution time. Not network traffic.
It's not supposed to be calculating anything at all but loading missing resources
(resources that could have been static parts of the HTML instead of XMLHttpRequests)

**Comment by Grumag** [comment]
Yes, the issue is still relevant.  Switching tabs stops the admin page loading. This cannot be by design. This is not browser related.

**Comment by cmorrisonmvnt** [comment]
Can we get some motion on this ticket? It's a total blocker to getting work done efficiently.

**Comment by alexwatever** [comment]
We are still seeing this issue with v2.4.5-p1 and it's a real pain for some users, clients.

**Comment by engcom-Delta** [comment]
Hi @davirs ,

Verified the issue on 2.4-develop and it is reproducible.

Steps to reproduce:-

1.Login to Magento Admin
2.Navigate to Products
3.Open multiple edit product links in new tabs
4.Wait a few seconds for all loading indicators to stop spinning for new tabs
5.Switch to new tabs that were opened


When switching to the new tabs, the pages are not ready to go. They are not fully loaded and require a little more time for the product fields and information to populate.


Screen Recording 2024-05-14 at 3.58.21 PM.zip
